### PR TITLE
fix(bctheme): BCTHEME-36 Product List Options Do Not Have a Max Height

### PR DESCRIPTION
**Pull request by BC-tymurbiedukhin**
#### What?

On Stencil themes, product list options do not have a set max height. This affects merchants that add a large amount of items to their product list options as it increases the length of the product page and pushes the add to cart button far down the page.

#### Tickets / Documentation

https://jira.bigcommerce.com/browse/BCTHEME-36

#### Screenshots (if appropriate)

Screenshot 2020-07-01 at 15 11 44

ping @yurytut1993 @junedkazi

### CODE REVIEW

File: assets/scss/components/stencil/productView/_productView.scss [hunk 1]
```diff
@@ -372,6 +372,8 @@
     border: $productOptions-list-border;
     list-style: none;
     margin: 0;
+    max-height: 400px;
+    overflow-y: scroll;
```
Comment (golcinho) [inline assets/scss/components/stencil/productView/_productView.scss:376]: I think that scrollbar shouldn't be visible when items count less than height of dropdown.

`overflow-y: auto` - provides more flexible behavior 
```suggestion
    overflow-y: auto;
```

Comment (BC-tymurbiedukhin) [reply assets/scss/components/stencil/productView/_productView.scss]: Thank you

File: assets/scss/components/stencil/productView/_productView.scss [hunk 2]
```diff
@@ -372,6 +372,8 @@
     border: $productOptions-list-border;
     list-style: none;
     margin: 0;
+    max-height: 400px;
```
Comment (junedkazi) [inline assets/scss/components/stencil/productView/_productView.scss:375]: How does this look on mobile or tablets. Can we confirm before we merge this one.

Comment (BC-tymurbiedukhin) [reply assets/scss/components/stencil/productView/_productView.scss:375]: Screenshot 2020-07-06 at 11 33 01
Screenshot 2020-07-06 at 11 33 22

**Pull request merged**